core/src/test/java/apoc/schema/SchemasTest.java

```diff
@@ -843,9 +849,9 @@ public void testIndexesMoreLabels() {
                         assertEquals("NO FAILURE", r.get("failure"));
                         assertEquals(100d, r.get("populationProgress"));
                         assertEquals(1d, r.get("valuesSelectivity"));
-                        Assertions.assertThat(r.get("userDescription").toString())
+                        assertThat(r.get("userDescription").toString())
                                 .contains(
-                                        "name='index3', type='TEXT', schema=(:Person {name}), indexProvider='text-2.0' )");
+                                        "name='index3', type='TEXT', schema=(:Person {name}), indexProvider='text-3.0' )");
```

Comment:
Not sure it is make sense to assert on a specific index provider.

Reply:
Agree, but I'll leave that for the next time :).